Tests/APITesters/SwiftAPITester/SwiftAPITester/ConfigurationAPI.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ConfigurationAPI.swift
+//  SwiftAPITester
+//
+//  Created by Joshua Liebowitz on 5/6/22.
+//
+
+import Foundation
+import RevenueCat
+
+func checkConfigurationAPI() {
+    let configuration = Configuration
+        .configurationBuilder(withAPIKey: "")
+        .with(apiKey: "")
+        .with(appUserID: "")
+        .with(observerMode: false)
+        .with(userDefaults: UserDefaults.standard)
+        .with(dangerousSettings: DangerousSettings())
+        .with(networkTimeoutSeconds: 1)
+        .with(storeKit1TimeoutSeconds: 1)
+        .with(usesStoreKit2IfAvailable: false)
+        .build()
```

Comment:
The usage looks awesome :chefskiss:

It does somewhat make me wonder whether we should name the `configurationBuilder` method something that looks more RevenueCat-specific. Otherwise, in Swift, that line will look generic enough that it could totally have belonged to any SDK, and it might be confusing to devs maintaining apps that use RC

Reply:
True, I'll think a bit about this.